plugins/optimization-detective/storage/rest-api.php

```diff
@@ -46,6 +46,11 @@ function od_register_endpoint(): void {
 			'required'    => true,
 			'pattern'     => '^[0-9a-f]{32}$',
 		),
+		'eTag'                => array(
```

Comment:
See other comment about calling this rather `current_etag`.

Reply:
Done in 371e998.